src/ZoneServer/Network/PacketHandler.cs

```diff
@@ -1612,6 +1612,7 @@ public void CZ_MAP_REVEAL_INFO(IZoneConnection conn, Packet packet)
 		{
 			var mapId = packet.GetInt();
 			var visible = packet.GetBin(128);
+			var percentage = packet.GetFloat();
```

Comment:
It seems like we use the same struct for reading and sending this data in our binary templates, suggesting that there might be a second float after the percentage, just like in the sending part. Have you looked into that by any chance? Is a float missing here or are our templates incorrect?